docs/src/pages/users/getting-started/account/index.mdx

```diff
@@ -3,121 +3,20 @@ layout: '@layouts/UsersLayout.astro'
 title: Manage your account
 ---
 
-{/*TODO Use page/screen consistently*/}
+import OCCRPCallout from "@snippets/OCCRPCallout.astro";
 
-# How to create an account or reset your password
+# Manage your account
 
-<p class="lead">On this page, you will find instructions on how to create an account for OCCRP Aleph and how to reset your password.</p>
+<p class="lead">OCCRP Aleph is an Aleph instances operated by the OCCRP. Learn how to create an account for OCCRP Aleph or how to resset your password.</p>
```

Comment:
Instances should be instance

Reply:
Thanks for catching that, just fixed it.